test/examples/star-cors-example.test.ts

```diff
@@ -29,4 +29,32 @@ describe('Star CORS examples', () => {
 
     response.headers['access-control-allow-origin'].should.eql('http://localhost:3000');
   });
+
+  it('/api triggers cors policy', async () => {
+    const response = await supertest('localhost:3000')
+      .get('/api')
+      .set('origin', 'http://lvh.me.foreign.com')
+      .expect(200);
+
+    response.headers['access-control-allow-origin'].should.not.eql('http://lvh.me.foreign.com');
+    response.headers['access-control-allow-origin'].should.eql('localhost:3000');
+  });
+
+  it('/api/example returns access-control-allow-origin that contains the subdomain', async () => {
+    const response = await supertest('localhost:3000')
+      .get('/api/example')
+      .set('origin', 'http://some.localhost:3000')
+      .expect(200);
+
+    response.headers['access-control-allow-origin'].should.eql('http://some.localhost:3000');
+  });
+
+  it('/api/example blocks deep subdomains', async () => {
+    const response = await supertest('localhost:3000')
+      .get('/api/example')
+      .set('origin', 'http://some.very.deep.subdomain.localhost:3000')
+      .expect(200);
+
+    response.headers['access-control-allow-origin'].should.eql('localhost:3000');
+  });
```

Comment:
Can we add another one when this is supported if in allowedOrigins the configuration is *.localhost:3000?

Reply:
Done. Added '*.lvh.me' in config to not break other tests that rely on localhost